README.md

```diff
@@ -51,31 +48,39 @@ The application template has the following structure:
 
 ```
 config/             Configuration files.
+    common/
+    console/
+    environments/
+    web/
 docs/               Documentation.
 public/             Files publically accessible from the Internet.
     assets/         Published assets.
     index.php       Entry script.
 resources/          Application resources.
     assets/         Asset bundle resources.
     layout/         Layout view templates.
-    view/           View templates.
+    messages/
```

Comment:
Add descriptions for new folders.

Reply:
I don't know what description you like...